.github/workflows/pluto_build.yml

```diff
@@ -73,26 +74,22 @@ jobs:
 
       - name: Set build environment variables
         run: |
-          # replace dashes with underscores to create a valid postgres schema name
-          BUILD_ENGINE_SCHEMA=$(echo ${BUILD_ENGINE_SCHEMA} | tr - _)
-          echo "BUILD_ENGINE_SCHEMA=$BUILD_ENGINE_SCHEMA" >> "$GITHUB_ENV"
-          # set postgres schema search path to prioritize BUILD_ENGINE_SCHEMA
-          BUILD_ENGINE=${BUILD_ENGINE_DATABASE}?options=--search_path%3D${BUILD_ENGINE_SCHEMA},public
+          BUILD_ENGINE=${BUILD_ENGINE_SERVER}/${BUILD_ENGINE_DB}
           echo "BUILD_ENGINE=$BUILD_ENGINE" >> "$GITHUB_ENV"
 
-      - name: Plan Build
-        working-directory: ./
+      - name: Load data
+        working-directory: products/pluto
         run: |
-          python3 -m dcpy.builds.load pluto
+          python3 -m dcpy.builds.load --product pluto
```

Comment:
To @fvankrieken 's suggestion in the prior PR, can we ✂️ the `--product pluto` bit, and have `loads` take an optional path to a recipe file? It's a pretty trivial change, and would be nice to do it while it's still fresh. (And if we need the product name for some reason, it's in the recipe.yml itself.)

Reply:
so `template_build.yml` would look something like
```bash
defaults:
  run:
    shell: bash
    working-directory: products/template

...

python3 -m dcpy.builds.load
```
?

and `pluto_build.yml` would look something like
```bash
defaults:
  run:
    shell: bash
    working-directory: products/pluto

...

python3 -m dcpy.builds.load --recipe ${{ inputs.recipe_filename }}
```
?